app/create-feed-from-gtfs/service.js

```diff
@@ -1,6 +1,40 @@
 import Ember from 'ember';
 
 export default Ember.Service.extend({
+<<<<<<< HEAD
```

Comment:
Expected '}' to match '{' from line 3 and instead saw '<<'.<br>Unexpected use of '<<'.<br>Expected an identifier and instead saw '<<'.<br>'HEAD' is not defined.